src/api/app/assets/javascripts/webui/application.js

```diff
@@ -207,7 +207,7 @@ function change_role(obj) {
         complete: function () {
             $('#' + type + '_spinner').hide();
             $('#' + type + '_table input').animate({opacity: 1}, 200);
-            $('#' + type + '_table input').removeAttr('disabled');
+            $('#' + type + '_table input').prop('disabled', false);
```

Comment:
I meant to say that removeAttr('disabled') already fails in jquery 2.1 - but build.oo uses jquery 1.11.

So :+1: for the change